open_instruct/utils.py

```diff
@@ -67,6 +69,29 @@
 
 DataClassType = NewType("DataClassType", Any)
 
+
+def ray_get_with_progress(ray_refs: List[ray.ObjectRef], desc: str = "Processing") -> List[Any]:
```

Comment:
nit: can we make the bar configurable a bit, e.g. only plays on a verbose mode? Looking at your wandb runs, it seems like then we log for e.g. every weight sync, which is useful for debugging but clutters up the log a bunch.

It also might be a bit weird at times, for example, the weight sync often looks longer than it actually is as it sits there and waits for the last generations to finish (although right now this doesnt quite seem to be the case, see previous review comment), which makes the loading bar kinda look funky (sitting around for a few minutes at 0%, then makes progress).

Reply:
discuss offline. 
I think making "Resetting vLLM prefix cache" and "Broadcasting weights to vLLM" only viewable in debug mode is best. Mixed feelings about the generation bar, probably its just useful to keep?